config_builder.go

```diff
@@ -1220,7 +1220,7 @@ func (d *DefaultDatabaseBuilder) BuildDatabase(
 	//
 	// TODO(ziggie): Add support for SQL payments DB.
 	// Mount the payments DB for the KV store.
-	paymentsDBOptions := []paymentsdb.OptionModifier{
+	paymentsDBOptions := []paymentsdb.StoreOptionModifier{
```

Comment:
i think it's ok to keep as just OptionModifier given that this is in the db package and so Store is implied. 

The reason that it was named more specifically in the graph package is cause there, we have 2 separate modifier types, one for the Store struct and then another for the ChannelGraph struct